src/components/buttons.js

```diff
@@ -0,0 +1,8 @@
+export function Button({prevPage, nextPage, setCurrentPageUrl}){
+  return(
+    <div>
+      {prevPage && <button className='btn' onClick={() => setCurrentPageUrl(prevPage)}>Previous</button>}
```

Comment:
Where does the `btn` class' style lives and why in `App.css` instead of `Buttons.css`?